.github/ISSUE_TEMPLATE/01_request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "❗ Request"
+about: A request for a template change, new feature, ...
```

Comment:
Request should probably be redirected to [Github discussions](https://docs.github.com/en/discussions)